web/themes/custom/bglobal/source/_patterns/01-molecules/hero/hero.twig

```diff
@@ -0,0 +1,22 @@
+<section class="hero__container">
+  <div class="hero__image__container">
+    <img class="hero__image" src="{{ hero.img.src }}" alt="{{ hero.img.alt }}">
+  </div>
+  <div class="hero__text__container container">
+    <h1 class="hero__title h2">
+      <span class="highlight--blue">{{ hero.title }}</span>
+    </h1>
+    {% if hero.subtitle %}
+    <h2 class="hero__subtitle p">
```

Comment:
`h2` tag should not have `p` as class here. We should probably use `h3` tag, or maybe `p` tag here (in which case, we can leave the `p` class.).